dev-tools/packaging/package_test.go

```diff
@@ -382,6 +384,31 @@ func checkDockerEntryPoint(t *testing.T, p *packageFile, info *dockerInfo) {
 	})
 }
 
+func checkDockerLabels(t *testing.T, info *dockerInfo, file string) {
+	if vendor := info.Config.Labels["org.label-schema.vendor"]; vendor != "Elastic" {
+		t.Errorf("unexpected vendor label: %s", vendor)
+	}
+
+	expectedLicense := "Elastic License"
+	ossPrefix := strings.Join([]string{
+		info.Config.Labels["org.label-schema.name"],
+		"oss",
+		info.Config.Labels["org.label-schema.version"],
+	}, "-")
+	if strings.HasPrefix(filepath.Base(file), ossPrefix) {
+		expectedLicense = "ASL 2.0"
+	}
+	if license, present := info.Config.Labels["license"]; !present || license != expectedLicense {
+		t.Errorf("unexpected license label: %s", license)
+	}
+}
+
+func checkDockerUser(t *testing.T, info *dockerInfo) {
+	if info.Config.User == "root" {
+		t.Errorf("unexpected docker user: %s", info.Config.User)
+	}
```

Comment:
I think auditbeat uses root.

Reply:
Bummer, it does, I guess package tests aren't run on pull requests?  I'll look for a workaround for this.

Reply:
counting on the fact that root ownership means run as root